opencti-platform/opencti-graphql/src/utils/authorizedMembers.ts

```diff
@@ -29,3 +40,37 @@ export const getAuthorizedMembers = async (
   });
   return authorizedMembers;
 };
+
+const hasExplorationCapabilities = (u: AuthUser) => {
+  const userCapabilities = u.capabilities.map((n) => n.name);
+  return userCapabilities.includes(BYPASS)
+    || (userCapabilities.includes('EXPLORE_EXUPDATE_EXDELETE') && userCapabilities.includes('EXPLORE_EXUPDATE'));
+};
+
+export const containsValidAdmin = async (
+  context: AuthContext,
+  authorized_members: Array<AuthorizedMember>,
+) => {
+  const adminIds = authorized_members
+    .filter((n) => n.access_right === MEMBER_ACCESS_RIGHT_ADMIN)
+    .map((e) => e.id);
+  if (adminIds.includes(MEMBER_ACCESS_ALL)) { // everyone is admin
+    return true;
+  }
```

Comment:
maybe add a condition here that checks that there is at least one member with admin right, before trying to resolve the admin ids, otherwise return false 
like this :
```
if (adminIds.length == 0) { 
   return false 
} 
```

It's the condition we had before.